system/keyboardlayouts/german.xml

```diff
@@ -9,13 +9,33 @@ Default font lacks support for all characters
       <row>1234567890ß</row>
       <row>qwertzuiopü</row>
```

Comment:
maybe add `+` as last button (and the other ones in the rows above and below), than it looks more like the real keyboard